### ISSUE TITLE
Trap throwing speed is unimplemented

### ISSUE THREAD

**Issue by disastranagant** [Openarl/PathOfBuilding#614]
And incorrectly uses cast speed.

**Comment by Openarl** [comment]
I'd love to implement it, but I have no idea what the base throwing speed is, as it doesn't seem to have been made public by GGG at any point. The same goes for mine laying speed and totem placement speed.

**Comment by aggixx** [comment]
I tested this on a character with no passive tree or equipment:

I threw 21 traps back to back, measuring the time between the first frame that the first trap was missing a charge to the first frame that the last trap had 0 remaining charges (20 trap throws worth of time). At 60 FPS it took 600 frames, or exactly 500ms per trap.

https://gfycat.com/NiftyFlickeringDipper

Then I placed 21 totems back to back, again measuring 20 totem placements worth of time. I used a decoy totem for the first and last totem so I could clearly measure when the totem placement of each begins. At 60 FPS it took 718 frames, or 598ms per placement. Actual base time is probably 600ms.

https://gfycat.com/EasyMajorHyena

Then I placed 21 mines back to back. I again started and finished with a single freeze mine. Here I measured from the time the first freeze mine appeared on the buff bar, to the time the last freeze mine appeared on the buff bar. At 60 FPS it took 599 frames, or 499ms per placement. Actual base time is probably 500ms.

https://gfycat.com/SecretSeparateAfricanparadiseflycatcher
